src/main/java/net/dv8tion/jda/core/entities/impl/ReceivedMessage.java

```diff
@@ -149,6 +149,66 @@ else if (reaction.isSelf())
         return channel.addReactionById(getIdLong(), unicode);
     }
 
+    @Override
+    public RestAction<Void> removeReaction(Emote emote)
+    {
+        Checks.notNull(emote, "Emote");
+
+        MessageReaction reaction = reactions.parallelStream()
```

Comment:
you should be using a normal stream

Reply:
I've copied this from the addReaction code, should I change it there too?

Reply:
Yes, please do.